tasks/lint_rules/src/main.rs

```diff
@@ -0,0 +1,94 @@
+use oxc_allocator::Allocator;
+use oxc_ast::{ast::ObjectPropertyKind, syntax_directed_operations::PropName, AstKind};
+use oxc_parser::Parser;
+use oxc_semantic::SemanticBuilder;
+use oxc_span::SourceType;
+use std::{fs::read_dir, path::Path};
+use ureq::Response;
+
+// TODOs:
+// - Error handling, message
+//   - JS parsing may fail?
+// - Use Result properly
+// - Support other rules
+// - Better AST traversal... :(
+
+const ORIGINAL: &str = "https://raw.githubusercontent.com/eslint/eslint/main/packages/js/src/configs/eslint-recommended.js";
+const OURS_DIR: &str = "crates/oxc_linter/src/rules/eslint";
```

Comment:
We need to import the linter crate so we can get more metadata

Reply:
Sorry, could you elaborate on this?